homeassistant/components/zha/core/cluster_handlers/general.py

```diff
@@ -553,6 +553,11 @@ class OtaClientClusterHandler(ClientClusterHandler):
         Ota.AttributeDefs.current_file_version.name: True,
     }
 
+    @callback
+    def attribute_updated(self, attrid: int, value: Any, timestamp: Any) -> None:
+        """Handle an attribute updated on this cluster."""
+        ClusterHandler.attribute_updated(self, attrid, value, timestamp)
```

Comment:
Optional, but maybe add a comment why we're skipping the `ClientClusterHandler` `attribute_updated` method here (=> to not create logbook events), so it's clear if somebody else looks at this.